toolchains/esp32/sdkconfig.defaults

```diff
@@ -1,70 +1,61 @@
-# CONFIG_APP_COMPILE_TIME_DATE is not set
+# This file was generated using idf.py save-defconfig. It can be edited manually.
+# Espressif IoT Development Framework (ESP-IDF) Project Minimal Configuration
+#
 CONFIG_BOOTLOADER_LOG_LEVEL_NONE=y
 CONFIG_BOOTLOADER_APP_ROLLBACK_ENABLE=y
 CONFIG_BOOTLOADER_SKIP_VALIDATE_IN_DEEP_SLEEP=y
+# CONFIG_APP_COMPILE_TIME_DATE is not set
 CONFIG_ESPTOOLPY_FLASHSIZE_4MB=y
 CONFIG_PARTITION_TABLE_CUSTOM=y
 CONFIG_COMPILER_OPTIMIZATION_SIZE=y
 CONFIG_COMPILER_OPTIMIZATION_ASSERTIONS_DISABLE=y
 CONFIG_BT_ENABLED=y
-CONFIG_BTDM_CTRL_BLE_MAX_CONN=2
-CONFIG_BTDM_CTRL_PINNED_TO_CORE_1=y
-# CONFIG_BTDM_CTRL_MODEM_SLEEP is not set
-# CONFIG_BTDM_BLE_SCAN_DUPL is not set
-CONFIG_BTDM_BLE_ADV_REPORT_FLOW_CTRL_NUM=50
 CONFIG_BT_NIMBLE_ENABLED=y
 CONFIG_BT_NIMBLE_MEM_ALLOC_MODE_DEFAULT=y
 CONFIG_BT_NIMBLE_LOG_LEVEL_NONE=y
 CONFIG_BT_NIMBLE_MAX_CONNECTIONS=2
 CONFIG_BT_NIMBLE_MAX_BONDS=4
 CONFIG_BT_NIMBLE_MAX_CCCDS=3
 CONFIG_BT_NIMBLE_PINNED_TO_CORE_1=y
-CONFIG_BT_NIMBLE_ROLE_PERIPHERAL=y
 CONFIG_BT_NIMBLE_NVS_PERSIST=y
 CONFIG_BT_NIMBLE_SVC_GAP_DEVICE_NAME="toit"
 CONFIG_BT_NIMBLE_SVC_GAP_APPEARANCE=0x0
 CONFIG_BT_NIMBLE_ACL_BUF_COUNT=10
-# CONFIG_SPI_SLAVE_ISR_IN_IRAM is not set
-CONFIG_UART_ISR_IN_IRAM=y
-CONFIG_ESP32_DEFAULT_CPU_FREQ_240=y
-# CONFIG_ESP32_SPIRAM_SUPPORT is not set
-# CONFIG_SPIRAM_IGNORE_NOTFOUND is not set
-# CONFIG_SPIRAM_USE_CAPS_ALLOC is not set
-# CONFIG_SPIRAM_CACHE_LIBNUMPARSER_IN_IRAM is not set
-# CONFIG_SPIRAM_CACHE_LIBCHAR_IN_IRAM is not set
-# CONFIG_SPIRAM_CACHE_LIBSTR_IN_IRAM is not set
-# CONFIG_ESP32_DEBUG_OCDAWARE is not set
-# CONFIG_ESP32_BROWNOUT_DET is not set
-CONFIG_ESP32_XTAL_FREQ_AUTO=y
+CONFIG_BTDM_CTRL_BLE_MAX_CONN=2
+CONFIG_BTDM_CTRL_PINNED_TO_CORE_1=y
+# CONFIG_BTDM_CTRL_MODEM_SLEEP is not set
+# CONFIG_BTDM_BLE_SCAN_DUPL is not set
+CONFIG_BTDM_BLE_ADV_REPORT_FLOW_CTRL_NUM=50
 # CONFIG_ADC_CAL_EFUSE_TP_ENABLE is not set
 # CONFIG_ADC_CAL_LUT_ENABLE is not set
+# CONFIG_SPI_SLAVE_ISR_IN_IRAM is not set
+CONFIG_UART_ISR_IN_IRAM=y
 # CONFIG_ESP_ERR_TO_NAME_LOOKUP is not set
 CONFIG_ETH_SPI_ETHERNET_DM9051=y
 CONFIG_ETH_SPI_ETHERNET_W5500=y
 CONFIG_ETH_USE_OPENETH=y
 # CONFIG_ESP_HTTP_CLIENT_ENABLE_HTTPS is not set
 CONFIG_ESP_SLEEP_GPIO_RESET_WORKAROUND=y
-CONFIG_ESP_IPC_TASK_STACK_SIZE=1024
-# CONFIG_ESP_NETIF_TCPIP_ADAPTER_COMPATIBLE_LAYER is not set
+CONFIG_XTAL_FREQ_AUTO=y
 # CONFIG_ESP_PHY_CALIBRATION_AND_DATA_STORAGE is not set
+CONFIG_ESP_DEFAULT_CPU_FREQ_MHZ_240=y
 # CONFIG_ESP_INT_WDT is not set
-# CONFIG_ESP_TASK_WDT is not set
-CONFIG_ESP32_WIFI_CACHE_TX_BUFFER_NUM=16
+# CONFIG_ESP_TASK_WDT_INIT is not set
+# CONFIG_ESP_DEBUG_OCDAWARE is not set
+# CONFIG_ESP_BROWNOUT_DET is not set
 # CONFIG_ESP32_WIFI_NVS_ENABLED is not set
 CONFIG_ESP32_WIFI_TASK_PINNED_TO_CORE_1=y
 # CONFIG_ESP32_WIFI_IRAM_OPT is not set
-# CONFIG_FMB_CONTROLLER_SLAVE_ID_SUPPORT is not set
-CONFIG_FMB_TIMER_PORT_ENABLED=y
```

Comment:
These seem to have disappeared on the right.

Reply:
This is something to do with modbus - it's not there any more in the new esp-idf.